qiita_pet/handlers/download.py

```diff
@@ -185,6 +186,9 @@ def get(self, filepath_id):
             self.set_header('Content-Type', 'application/octet-stream')
             self.set_header('Content-Transfer-Encoding', 'binary')
             self.set_header('X-Accel-Redirect', '/protected/' + relpath)
+            aid = filepath_id_to_object_id(fid)
+            if aid is not None:
+                fname = '%d_%s' % (aid, fname)
```

Comment:
do we intend to split aid and fname somewhere else according to the `_` char? If so, should be allow further `_` in `fname`?

Reply:
No, we don't; this is just so users have the artifact id prepended in the name, [as you suggested](https://github.com/biocore/qiita/issues/2174#issuecomment-398388959) ...